core/model/modx/rest/modrestservice.class.php

```diff
@@ -59,6 +59,7 @@ public function __construct(modX &$modx,array $config = array()) {
             'responseSuccessKey' => 'success',
             'trimParameters' => false,
             'xmlRootNode' => 'response',
+            'xmlDisableEntityLoader' => true,
```

Comment:
Since the setting only works for a specific version range of the library, do we need this as default setting?